src/yarp.c

```diff
@@ -1837,19 +1861,20 @@ yp_instance_variable_write_node_create(yp_parser_t *parser, yp_instance_variable
 
 // Allocate a new InterpolatedRegularExpressionNode node.
 static yp_interpolated_regular_expression_node_t *
-yp_interpolated_regular_expression_node_create(yp_parser_t *parser, const yp_token_t *opening, const yp_token_t *closing) {
+yp_interpolated_regular_expression_node_create(yp_parser_t *parser, const yp_token_t *opening) {
   yp_interpolated_regular_expression_node_t *node = yp_alloc(parser, sizeof(yp_interpolated_regular_expression_node_t));
 
   *node = (yp_interpolated_regular_expression_node_t) {
     {
       .type = YP_NODE_INTERPOLATED_REGULAR_EXPRESSION_NODE,
       .location = {
         .start = opening->start,
-        .end = closing->end
+        .end = opening->end
```

Comment:
Nit but I wonder if we should set `.end = NULL` here. It should be overwritten, but in the off chance this method is ever called without a subsequent `yp_interpolated_regular_expression_node_closing_set` call, it'd be more clear to have the end as NULL